core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -395,17 +417,18 @@ private static String formatFragment(
                 })
                 .collect(toImmutableList());
         builder.append(indentString(1));
+        String hashColumn = partitioningScheme.getHashColumn().map(column -> "[" + column + "]").orElse("");
```

Comment:
I've left this because formatFragment is only used for `EXPLAIN TEXT DISTRIBUTED`. It isn't part of any anonymisation code path. Once we implement `EXPLAIN ANONYMIZE`, we can take a look at this.

Reply:
Added todo at the start of method